src/contexts/LanguageContext.tsx

```diff
@@ -39,10 +37,9 @@ export const LanguageProvider: React.FC<LanguageProviderProps> = ({ children })
     setLanguageState(lang);
   };
 
-  // Translation function with English fallback
   const t = (key: TranslationKey): string => {
     const translation = translations[language][key];
-    return translation || translations['en'][key] || key;
+    return translation || translations['ar'][key] || key;
```

Comment:
We can avoid the magic number by creating a constant of the default language, something like:
```js
return translation || translations[DEFAULT_LANGUAGE][key] || key;
```

See:
- [Replace Magic Literal - Refactoring](https://www.refactoring.com/catalog/replaceMagicLiteral.html)
- [Magic Numbers](https://clang.llvm.org/extra/clang-tidy/checks/readability/magic-numbers.html)

Reply:
Not a blocking comment, will do the refactoring after the merge.